resources/assets/tconstruct/lang/zh_CN.lang

```diff
@@ -0,0 +1,832 @@
+tile.tconstruct.slime_vine_blue_end.name=蓝色史莱姆藤蔓
+tile.tconstruct.slime_vine_purple.name=紫色史莱姆藤蔓
+tile.tconstruct.slime_vine_purple_mid.name=紫色史莱姆藤蔓
+tile.tconstruct.slime_vine_purple_end.name=紫色史莱姆藤蔓
+
+tile.tconstruct.seared.stone.name=焦黑石头
+tile.tconstruct.seared.cobble.name=焦黑圆石
+tile.tconstruct.seared.paver.name=焦黑盖板石
+tile.tconstruct.seared.brick.name=焦黑石砖
+tile.tconstruct.seared.brick_cracked.name=裂纹焦黑石砖
+tile.tconstruct.seared.brick_fancy.name=平滑焦黑石砖
+tile.tconstruct.seared.brick_square.name=錾制焦黑石砖
+tile.tconstruct.seared.brick_triangle.name=三角纹焦黑石砖
+tile.tconstruct.seared.brick_small.name=细纹焦黑石砖
+tile.tconstruct.seared.road.name=焦黑铺路石
+tile.tconstruct.seared.tile.name=焦黑瓷砖
+tile.tconstruct.seared.creeper.name=焦黑雕琢石砖块
+tile.tconstruct.seared.tooltip=既可以做冶炼炉的材料，也是一种美观的建材
+
+tile.tconstruct.seared_slab.stone.name=焦黑石头台阶
+tile.tconstruct.seared_slab.cobble.name=焦黑圆石台阶
+tile.tconstruct.seared_slab.paver.name=焦黑盖板石台阶
+tile.tconstruct.seared_slab.brick.name=焦黑石砖台阶
+tile.tconstruct.seared_slab.brick_cracked.name=裂纹焦黑石砖台阶
+tile.tconstruct.seared_slab.brick_fancy.name=平滑焦黑石砖台阶
+tile.tconstruct.seared_slab.brick_square.name=錾制焦黑石砖台阶
+tile.tconstruct.seared_slab.road.name=焦黑铺路石台阶
+tile.tconstruct.seared_slab2.creeper.name=焦黑雕琢台阶
+tile.tconstruct.seared_slab2.brick_triangle.name=三角纹焦黑石砖台阶
+tile.tconstruct.seared_slab2.brick_small.name=细纹焦黑石砖台阶
+tile.tconstruct.seared_slab2.tile.name=焦黑瓷砖台阶
+
+tile.tconstruct.seared_stairs_stone.name=焦黑石头楼梯
+tile.tconstruct.seared_stairs_cobble.name=焦黑圆石楼梯
+tile.tconstruct.seared_stairs_paver.name=焦黑盖板石楼梯
+tile.tconstruct.seared_stairs_brick.name=焦黑石砖楼梯
+tile.tconstruct.seared_stairs_brick_cracked.name=裂纹焦黑石砖楼梯
+tile.tconstruct.seared_stairs_brick_fancy.name=平滑焦黑石砖楼梯
+tile.tconstruct.seared_stairs_brick_square.name=錾制焦黑石砖楼梯
+tile.tconstruct.seared_stairs_brick_triangle.name=三角纹焦黑石砖楼梯
+tile.tconstruct.seared_stairs_brick_small.name=细纹焦黑石砖楼梯
+tile.tconstruct.seared_stairs_road.name=焦黑铺路石楼梯
+tile.tconstruct.seared_stairs_tile.name=焦黑瓷砖楼梯
+tile.tconstruct.seared_stairs_creeper.name=焦黑雕琢楼梯
+
+tile.tconstruct.smeltery_controller.name=冶炼炉控制器
+tile.tconstruct.smeltery_controller.tooltip=相当于冶炼炉的大脑
+tile.tconstruct.smeltery_io.drain.name=冶炼炉排液口
+tile.tconstruct.smeltery_io.drain.tooltip=可以向冶炼炉输入或输出液体
+tile.tconstruct.casting.table.name=浇铸台
+tile.tconstruct.casting.basin.name=铸造盆
+tile.tconstruct.casting.tooltip=基础铸造设备
+tile.tconstruct.faucet.name=浇注口
+tile.tconstruct.faucet.tooltip=让冶炼炉的内容物流出来
+tile.tconstruct.seared_tank.tank.name=焦黑储罐
+tile.tconstruct.seared_tank.gauge.name=焦黑容器
+tile.tconstruct.seared_tank.window.name=焦黑视窗
+tile.tconstruct.seared_tank.tooltip=敲掉时也能保存其中的流体
+tile.tconstruct.seared_glass.glass.name=焦黑玻璃
+tile.tconstruct.seared_glass.glass.tooltip=可以做冶炼炉的材料，也是美观的建材\n不能存储液体
+
+tile.tconstruct.stone_torch.name=石柄火把
+tile.tconstruct.stone_ladder.name=石头楼梯
+tile.tconstruct.wood_rail.name=木质轨道
+tile.tconstruct.wood_rail_dropper.name=带活版门的木质轨道
+tile.tconstruct.wood_rail_dropper.tooltip=漏斗矿车可以\n穿过这个方块下面

```

Comment:
Line 175 and 176 should be "tile.tconstruct.wood_rail_trapdoor.name" and "tile.tconstruct.wood_rail_trapdoor.tooltip"